index.html

```diff
@@ -4,6 +4,11 @@
     <title>The Best Recipes!</title>
   </head>
   <body>
-    <h1>Recipes</h1>
+    <h1>My Favorite Recipes</h1>
+    <h2>Recipes by category</h2>
+    <ul>
+      <h3><li><a href="singaporean.html">Singaporean Cuisine</a></li></h3>
```

Comment:
i find the choice of h3 within a list a bit odd. Maybe if you need it bold we could use `<strong>` or `<b>` here instead, depending on the importance? Strong signifies more importance than bold